src/components/admin-table/admin-table.tsx

```diff
@@ -39,8 +54,27 @@ export default function AdminTable() {
     React.useState<VisibilityState>({ recipientEmail: false });
   const [expanded, setExpanded] = React.useState<ExpandedState>({});
 
+  const [reimbursements, setReimbursements] = React.useState<Reimbursement[]>(
+    [],
+  );
+  const [isLoading, setIsLoading] = React.useState<boolean>(true);
+  const [error, setError] = React.useState<Error | null>(null);
+
+  React.useEffect(() => {
+    fetchReimbursements()
+      .then((data) => {
+        setReimbursements(data);
+        setIsLoading(false);
+        console.log(reimbursements);
+      })
+      .catch((err) => {
+        setError(err);
+        setIsLoading(false);
+      });
+  }, [reimbursements]);
```

Comment:
setting the dependency arrray to `[reimbursements]` will cause an infinite loop, since whenever `reimbursements` is updated, it will cause another update of `reimbursements`. we only need to fetch reimbursements on the component's initial render, so we can instead set the dep array to `[]`